install-dev/data/db_structure.sql

```diff
@@ -393,7 +393,6 @@ CREATE TABLE `PREFIX_category_product` (
 CREATE TABLE `PREFIX_cms` (
   `id_cms` int(10) unsigned NOT NULL AUTO_INCREMENT,
   `id_cms_category` int(10) unsigned NOT NULL,
-  `position` int(10) unsigned NOT NULL DEFAULT '0',
```

Comment:
We can't delete SQL field like that, it's a huge BC, if people are using it, everything will fall in error :(